i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md

```diff
@@ -140,17 +140,45 @@ By:
 - Move to PHP 8.0
 - Preparing Debian 11 support
 
-## Centreon Engine
+##  Centreon Collect
+
+> As of the 21.10.0 version, the components of Centreon Collect (Centreon Broker, Centreon Clib, Centreon Engine and Centreon Connectors)
+> are released concurrently.  They are now grouped under this section.
+
+### 21.10.1
+
+Release date: `February 23, 2022`
+
+#### Centreon Broker
+
+##### Improvements
+- Improved the multiplexing of events, which was a performance bottleneck. The processing speed of queued events should be significantly increased.
+- Some TLS trace logs were logged as errors, flooding the log file.
+- The SQL connections stats have been added to the broker stats available via gRPC.
+- Added the SQL Manager statistics in gRPC and stabilized the stats provider.
+- Added to LUA Stream Connectors' perfdata label parser the compatibility with centreon-plugins new metrics naming convention.
+
+##### Bug fixes
+
+- Fixed a regression due to the central broker's cache generation optimization, which was too thorough and prevented BAM from computing KPIs based on boolean rules
+- The central broker's cache generation loaded too much data and took too much time when BAM was activated.
+- Fixed an issue that could cause segmentation faults in centreon-engine when scheduling external commands
+- Fixed a design issue to avoid trying to access variables of broker's new logger when the logger is stopped. This issue could cause segmentation faults.
```

Comment:
```suggestion
- Fixed a design issue to avoid trying to access variables of broker's new logger when the logger was stopped. This issue could cause segmentation faults.
```